CEXT-4921: clarify event configuration step (#410)
AdobeDocs/commerce-extensibility: src/pages/starter-kit/checkout/configure.md

## Configure Eventing

> <InlineAlert variant="info" slots="text"/>
> 
> This step is required only if you want to use the [Eventing feature](https://developer.adobe.com/commerce/extensibility/events/). Skip this step if you do not plan to use the Eventing feature.
> For the use case of third-party events processing, refer to [Third-party events processing](./use-cases.md#third-party-events-processing)

Reviewer: ```suggestion
<InlineAlert variant="info" slots="text, text1"/>

The following configuration is required if you want to use the [Eventing feature](https://developer.adobe.com/commerce/extensibility/events/). Skip this process if you do not plan to use the Eventing feature.
For third-party events processing, refer to [Third-party events processing](./use-cases.md#third-party-events-processing)
```

    - `COMMERCE_ADOBE_IO_EVENTS_MERCHANT_ID`: The merchant ID of the Commerce instance.
    - `COMMERCE_ADOBE_IO_EVENTS_ENVIRONMENT_ID`: The environment ID of the Commerce instance.

> 1. Configure your [app.config.yaml](#appconfigyaml) file to register the events to consume by the app.

Reviewer: ```suggestion
1. Configure your [app.config.yaml](#appconfigyaml) file to register the events for the app to consume.
```

>     - Update `events_of_interest` field with the events that your app is interested in.
>     - Update `runtime_action` field with the action that will consume the events.

Reviewer: ```suggestion
    - Update the `events_of_interest` field with the events that your app requires.
    - Update the `runtime_action` field with the action that will consume the events.
```

> This step must be completed before deploying the application for event registration.

Reviewer: ```suggestion
This process must be completed before deploying the application for event registration.
```

| `events_metadata`   | List of event metadata to register. (This is not required for the Commerce event provider, since it uses event subscriptions.)|
| `subscription`      | Only required for the Commerce event provider. List of Commerce events to subscribe to. For payload specifications, refer to [Subscribe to events](../../events/api.md#subscribe-to-events).|

### app.config.yaml

The [`app.config.yaml`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/app.config.yaml) file defines the events of interest for your app. The events will be registered during [app deployment](./development.md#deploy-the-application).

```yaml
  events:
    registrations:
      Commerce events consumer:
        description: Consumes events from Adobe Commerce
        events_of_interest:
          - provider_metadata: dx_commerce_events
            event_codes:
              - com.adobe.commerce.observer.sales_order_creditmemo_save_after
        runtime_action: commerce-checkout-starter-kit/consume
```

> Check out the [App Builder with I/O events](https://developer.adobe.com/events/docs/guides/appbuilder/) guide for more information on how the events are registered in the app.

Reviewer: ```suggestion
Refer to the [App Builder with I/O events](https://developer.adobe.com/events/docs/guides/appbuilder/) guide for more information on how to register events in the app.
```
